src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -611,6 +799,17 @@ class ExternalConnectionHandler : public grpc_core::TcpServerFdHandler {
 
   // TODO(yangg) resolve duplicate code with on_read
   void Handle(int listener_fd, int fd, grpc_byte_buffer* buf) override {
+    if (grpc_event_engine::experimental::UseEventEngineServer()) {
+      grpc_event_engine::experimental::SliceBuffer pending_data;
+      if (buf != nullptr) {
+        pending_data =
+            grpc_event_engine::experimental::SliceBuffer::TakeCSliceBuffer(
+                buf->data.raw.slice_buffer);
+      }
+      (void)s_->ee_listener->HandleExternalConnection(listener_fd, fd,
```

Comment:
Why are we burying this potential error?

Reply:
Logged this error now.